tests/api/search/search_languages_entities.test.js

```diff
@@ -0,0 +1,42 @@
+import { getByUris } from '#tests/api/utils/entities'
+import { search, waitForIndexation } from '#tests/api/utils/search'
+import 'should'
+
+const wikidataUris = [ 'wd:Q10134', 'wd:Q19852850', 'wd:Q33111' ]
+
+describe('entities:languages:search', () => {
+  before(async () => {
+    // Ensure wikidata uris are indexed in the current format
+    await getByUris(wikidataUris, null, true)
+    await Promise.all(wikidataUris.map(uri => waitForIndexation('wikidata', uri.split(':')[1])))
```

Comment:
What about reusing `findOrIndexEntities`?

```suggestion
    await findOrIndexEntities(wikidataUris)
```

Reply:
Indeed! Done in 85e699396